### ISSUE TITLE
License requirements

### ISSUE THREAD

**Issue by fmalykh** [jfrog/terraform-provider-xray#51]
**Describe the bug**
Hi there,
We have a `Commercial` type Artifactory license with X-Ray functionality up and running:

```
{
  "type" : "Commercial",
  "validThrough" : "Apr 22, 2023",
  "licensedTo" : "Redacted"
}
```
Are we eligible to use this terraform provider?

`Error: Artifactory Projects requires Enterprise license to work with Terraform!`

We have no issues using `https://github.com/jfrog/terraform-provider-artifactory` which allows setting `check_license = false`:
https://registry.terraform.io/providers/jfrog/artifactory/latest/docs#check_license

**Comment by chb0github** [comment]
@fmalykh - you can also disable license check

`check_license = false`

**Comment by danielmkn** [comment]
yup, I see the change. I don't use it in the Xray provider. I'll add `Commercial` to the list today.

**Comment by fmalykh** [comment]
@chb0github I tried before submitting this but it's an invalid option
```
terraform -version
Terraform v0.14.5
+ provider registry.terraform.io/jfrog/artifactory v6.7.2
+ provider registry.terraform.io/jfrog/xray v1.1.3

terraform plan

Error: Unsupported argument

  on main.tf line 10, in provider "xray":
  10:   check_license = false

An argument named "check_license" is not expected here.
```

**Comment by danielmkn** [comment]
Hold on, fix is on the way.

**Comment by chb0github** [comment]
@danielmkn - it would be better to just use that shared code since it's common to other providers and we're trying to get consistent with all of them

**Comment by danielmkn** [comment]
@fmalykh, released in 1.1.4!

**Comment by fmalykh** [comment]
worked like a charm! lightning fast! thanks!
